portfolio/src/main/webapp/style.css

```diff
@@ -37,4 +37,44 @@ hr.bar {
 .main-pic {
   width: 50%;
   height: 50%;
+}
+
+table.wics {
+  margin-top: 20px;
+  text-align: left;
+  font-size: 25px;
+}
+ 
+.wics-note {
+  margin-top: 20px;
+  max-width: 50%;
+  float: left;
+  font-size: 20px;
+}
+
+.wics-pic {
+  float: right;
+  margin-right: 75%;
+  max-height: 115%;
+  max-width: 115%;
+  margin-top: 10px;
+}
+
+hr.end-hobby {
+  overflow: visible;
+  padding: 0;
+  border: none;
+  border-top: medium double #333;
+  color: #333;
+  text-align: center;
+}
+
+hr.end-hobby:after {
+  content: "§";
```

Comment:
nit: use single (') instead of double (") quotation marks in CSS

https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks

Reply:
Thanks for catching this!